src/components/widgets/MetadataWidget/MetadataWidget.tsx

```diff
@@ -46,28 +46,27 @@ function MetadataWidget(props: MetadataWidgetProps) {
             <EuiFlexGroup direction="column">
               <EuiFlexItem>
                 <EuiFlexGroup>
-                  <EuiFlexItem grow={false}>
+                  <EuiFlexItem grow={false} style={{ maxWidth: 600 }}>
```

Comment:
those restrict the width of the MetadataWidget, if the description/title/iri are too large.